src/States/MainMenuState.cpp

```diff
@@ -74,6 +74,13 @@ void MainMenuState::initialize()
 	mFileIoDialog.anchored(false);
 	mFileIoDialog.hide();
 
+    Font* tiny_font = Utility<FontManager>::get().font(constants::FONT_PRIMARY, constants::FONT_PRIMARY_NORMAL);
+	auto& r = NAS2D::Utility<NAS2D::Renderer>::get();
+	lblVersion.font(tiny_font);
+	lblVersion.text(constants::VERSION);
+	lblVersion.position(r.width() - tiny_font->width(constants::VERSION) - 5, r.height() - tiny_font->height() - 5);
+	lblVersion.color(NAS2D::Color::White);
```

Comment:
Would be nice if a constructor could take some or all of these properties.

Reply:
I opted for cleaner c'tors. Again, not opposed though to be fair the C# notion of properties and setting them during object instantiation is awesome:

```csharp
var SomeObject = new SomeObjectType
{
    Property1 = Whatever;
    Property2 = Whatever;
    // ... etc.
};
```

If modern C++ has a similar notion of initializing an object in this way (I recall seeing similar syntax somewhere in the NAS2D code) I'd be completely okay doing it that way ... would make this a little cleaner at least versus calling a bunch of member methods after object instantiation.

To be fair though, it's a different change set and could be figured out later on.